evalml/automl/utils.py

```diff
@@ -207,29 +205,6 @@ def get_best_sampler_for_data(X, y, sampler_method, sampler_balanced_ratio):
             return "Undersampler"
 
 
-def get_hyperparameter_ranges(component_graph, custom_hyperparameters):
```

Comment:
Moved to `PipelineBase` so we don't have to use `ComponentGraph.linearized_component_graph`, which I deleted